pkgs/applications/video/mirakurun/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, stdenvNoCC
+, bash
+, common-updater-scripts
+, fetchFromGitHub
+, genericUpdater
+, jq
+, makeWrapper
+, mkYarnPackage
+, which
+, writers
+, v4l-utils
+, yarn
+, yarn2nix
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mirakurun";
+  version = "3.8.0";
+
+  src = fetchFromGitHub {
+    owner = "Chinachu";
+    repo = "Mirakurun";
+    rev = version;
+    sha256 = "1fmzi3jc3havvpc1kz5z16k52lnrsmc3b5yqyxc7i911gqyjsxzr";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  phases = [ "installPhase" ];
+
+  mirakurun = mkYarnPackage rec {
+    inherit pname version src;
+
+    yarnNix = ./yarn.nix;
+    yarnLock = ./yarn.lock;
+    packageJSON = ./package.json;
```

Comment:
did you make any changes to the package.json ?
Otherwise, it doesn't need to be commited.

Reply:
I brought this back because it made the evaluation checks unhappy for whatever reason.